### DISCUSSION TITLE
弃用警告

### DISCUSSION THREAD

**Discussion by lym0518** [ppoffice/hexo-theme-icarus#1064] [Q&A]
Deprecation warning: use moment.updateLocale(localeName, config) to change an existing locale. moment.defineLocale(localeName, config) should only be used for creating a new locale See http://momentjs.com/guides/#/warnings/define-locale/ for more info.

hexo生成时会出现该警告

**Comment by ppoffice** [comment]
@lym0518 目前暂无更新moment.js的计划。
